eclass/kernel-install.eclass

```diff
@@ -541,12 +542,12 @@ kernel-install_pkg_pretend() {
 
 	if ! use initramfs && ! has_version "${CATEGORY}/${PN}[-initramfs]"; then
 		ewarn
-		ewarn "WARNING: The standard configuration of the Gentoo distribution"
-		ewarn "kernels requires an initramfs! You have disabled the initramfs"
-		ewarn "USE flag and as a result dracut was not pulled in as a dependency."
-		ewarn "Please ensure that you are either overriding the standard"
-		ewarn "configuration or that an alternative initramfs generation plugin"
-		ewarn "is installed for your installkernel implementation!"
+		ewarn "WARNING: The standard Gentoo distribution kernel configuration"
+		ewarn "requires an initramfs to reliably boot! You have disabled the"
```

Comment:
`to reliably boot` -> `to boot`

Either it boots or it does not, reliable implies that it would only sometimes not boot

Reply:
I think that is the case, because the dist-kernel config doesn't guarantee modules required to boot are builtin, some things may become a module or builtin, so across versions, booting is unreliable without an initramfs. I think there are cases where dist-kernel may just work without an initramfs but this can't be relied on.

Reply:
> some things may become a module or builtin

We never really do this, the config we use is mostly upstream Fedora's config and I don't think they do this either.

If it boots without an initramfs then I expect it will continue to boot without one,  the issue is that it simply cannot boot most machines in the first place.